README.md

```diff
@@ -67,14 +72,14 @@ Running the game over internet currently requires setting up a [Photon Transport
 > Checkout our Photon-Realtime setup guide, here:
 > [Boss Room Photon Setup Guide](Documentation/Photon-Realtime/Readme.md)
 
-Alternatively you can use Port Forwarding. The wonderful https://portforward.com/ site has guides on how to enable port forwarding on a huge number of routers. Boss Room uses `UDP` and needs a `9998` external port to be open. 
+Alternatively you can use Port Forwarding. The https://portforward.com/ site has guides on how to enable port forwarding on a huge number of routers. Boss Room uses `UDP` and needs an `9998` external port to be open. 
```

Comment:
a `9998` is correct
```suggestion
Alternatively you can use Port Forwarding. The https://portforward.com/ site has guides on how to enable port forwarding on a huge number of routers. Boss Room uses `UDP` and needs a `9998` external port to be open. 
```

Reply:
we had a lot of thoughts on 'a' or 'an' do you ignore the numbers  or not ....im not sure